OpenRA.Game/Map/MapGenerationArgs.cs

```diff
@@ -0,0 +1,43 @@
+#region Copyright & License Information
+/*
+ * Copyright (c) The OpenRA Developers and Contributors
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using OpenRA.Primitives;
+
+namespace OpenRA
+{
+	public class MapGenerationArgs
+	{
+		[FieldLoader.Require]
+		public string Uid = null;
```

Comment:
Is this the generated map's Uid?
Similar for `Title` below.

Reply:
Yes. (I'm not sure if this is requesting any changes?)